src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -1823,6 +1826,29 @@ def load_arguments(self, _):
             arg_type=get_enum_type(node_os_skus_update),
             validator=validate_os_sku,
         )
+        # local dns
+        c.argument(
+            "set_localdns",
+            options_list=["--set-localdns"],
```

Comment:
It looks like this option functions as a switch. If that's the case, the convention in the AKS command group is to name such options using a pattern like `--enable-localdns`.

Reply:
We switched from having the switch to just having a singular command --localdns-config which expects a filename. Does that need any fixes to follow conventions too? 

Thanks for helping me learn the convention here :)